src/classBased/components/InputTodo.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from "react"
+
+class InputTodo extends Component {
+state = {
+    title: ""
+    };
+    onChange = e => {
+        this.setState({
+            title: e.target.value
+          });
+      };
+      handleSubmit = e => {
+        e.preventDefault()
+        if (this.state.title.trim()) {
+          this.props.addTodoProps(this.state.title)
+          this.setState({
+            title: "",
+          })
+        } else {
+          alert("Please write item")
+        }
+      }
```

Comment:
Please,  as stated by Microverse among the [JavaScript best practices ](https://github.com/microverseinc/curriculum-html-css/blob/bb99dfef44441b211ff7f1d92f1f621951545d32/articles/javascript_best_practices.md), avoid the use of the `alert`  statement in your code. This is because it is impossible to style, it stops code execution and displays differently on different browsers. Kindly use a custom modal instead.